locksmith/__tests__/controllers/userController.test.ts

```diff
@@ -0,0 +1,109 @@
+const request = require('supertest')
+const app = require('../../src/app')
+const models = require('../../src/models')
+import UserOperations = require('../../src/operations/userOperations')
```

Comment:
Any reason for the differences in import styles?

Reply:
good question, let me sort that out